Formula/wasmer.rb

```diff
@@ -13,13 +13,17 @@ class Wasmer < Formula
     sha256 cellar: :any_skip_relocation, mojave:        "ca3bb35344bf5e3269e123842f622180939b3b240f0e98f1824ca79305212cae"
   end
 
-  depends_on "cmake" => :build
+  depends_on "llvm" => :build
   depends_on "rust" => :build
-  depends_on "wabt" => :build
+  depends_on "libffi" => :build
```

Comment:
Are you sure? I think you're only risking creating runtime dependencies that will cause another audit error, but perhaps you know something I don't.

Reply:
Once the binary is generated there should be no runtime dependency over llvm, libffi or rust :)

Reply:
Nope, you picked up another runtime dep:
```
==> brew linkage --test wasmer
==> FAILED
Broken dependencies:
  /usr/local/opt/libffi/lib/libffi.7.dylib (libffi)
  /usr/local/opt/llvm/lib/libc++.1.dylib (llvm)
```